### PR TITLE
Improvements sales packages

### PR DESCRIPTION
**Pull request by pyotrritchie**
Just working on some tests to add in..

### CODE REVIEW

File: src/Service.Host/index.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Products (local hotload)</title>
+    </head>
+    <body>
+        <div id="root"></div>
+        <script type="text/javascript">
+            window.APPLICATION_SETTINGS = {
+                authorityUri: 'https://www-sys.linn.co.uk/auth/',
+                appRoot: 'http://localhost:5050',
```
Comment (richardiphillips) [inline src/Service.Host/index.html]: Why did you change this. Would only be 5050 if you were running locally in a docker?

File: src/Service/Modules/SalesPackageModule.cs [hunk 2]
```diff
@@ -56,14 +66,35 @@ private object GetSalesPackages()
...
-            return this.Negotiate.WithModel(result).WithMediaRangeModel("text/html", ApplicationSettings.Get)
+            return this.Negotiate.WithModel(result)
+                .WithMediaRangeModel("text/html", ApplicationSettings.Get)
                 .WithView("Index");
         }
+
+        private Expression<Func<SalesPackage, string>> GetSortExpressionOnProperty(string sortBy)
+        {
+            var param = Expression.Parameter(typeof(SalesPackage));
+            return Expression.Lambda<Func<SalesPackage, string>>(Expression.Property(param, sortBy), param);
```
Comment (richardiphillips) [inline src/Service/Modules/SalesPackageModule.cs]: Think I'd have to debug this to figure out what it means!

File: src/Service.Host/index.html (hunk 1 above)
Comment (pyotrritchie) [reply src/Service.Host/index.html]: I've reverted that back on a commit, coming to github soon. It was just the port that the server runs on when you run the service via the command line

File: src/Service/Modules/SalesPackageModule.cs (hunk 2 above)
Comment (pyotrritchie) [reply src/Service/Modules/SalesPackageModule.cs]: I'll put some documentation on it
